### ISSUE TITLE
Navigacija

### ISSUE THREAD

**Issue by blazkablatnik** [63150055/crypto-portal#9]
> (a) puscica na naslednji izziv naj bo še hkratni return (v primeru,
    da je bil odgovor pravilen)
    (b) morda bi lahko delovala tudi puscica v desno na tipkovnici
        in pa morebiti se poteg s prstom na tablici
    (c) povrsina, ki sprejme, da hocemo na nov izziv,
        bi verjetno lahko bila vecja

**Comment by blazkablatnik** [comment]
"poteg s prstom na tablici" in "povrsina, ki sprejme, da hocemo na nov izziv,
bi verjetno lahko bila vecja" bi se lahko popravila ob morebitni prilagoditvi celotnega portala za telefone. Takrat bi bilo smiselno popraviti funkcionalnosti, saj bi bil portal "namenjen" tudi uporabnikom mobilnih naprav.

**Comment by jaanos** [comment]
Hmm, meni sedaj ob vnosu pravilnega odgovora pri read/medium ne deluje niti Enter, niti klik na desno puščico ali desna puščica na tipkovnici. Pri read/hard je še vedno isto, torej je treba pritisniti Enter.

Poganjam lokalno, uporabljam Firefox 57 na Ubuntu.

**Comment by blazkablatnik** [comment]
Ja vidim ja. Nisem opazila, ker je Chrome veliko bolj prijazen. Tm sem vse testirala in meni so vsi popravki delali. Sem pa preverila za Mozillo in precej drugače deluje, pa trenutno ne razumem zakaj (popravila sem, da se vnos preveri ampak v kolikor je pravilen, se ob pritisku na return hkrati prikaže kar naslednja rešitev. Tega obnašanja si ne znam razložiti. Bom pogledala ko bo prilika.) Tisti fix je pa pushan.

Hvala 😊

**Comment by jaanos** [comment]
Z zadnjim commitom pri meni dela, na [razvojni različici](https://lkrv.fri.uni-lj.si/crypto-devel/) pa mi ne dela niti na laptopu niti na telefonu (Chrome na Androidu). Read/hard deluje na obeh. To me rahlo čudi - saj ni nič takega, kar bi predpostavljalo, da je aplikacija direktno pod rootom strežnika, kajne? V prometu ne vidim nič takšnega, v JS konzoli pa dobim tako pri sebi kot na strežniku eno napako takoj ob nalaganju strani (ne najde `imageRightFlag`), kasneje pa nič več.

Mimogrede: če ti razvojna različica portala ne deluje, je to najbrž zato, ker je pred kratkim potekel korenski certifikat @*fri*CA in sem zato moral generirati novega. Da bo delovala, odstrani trenutni korenski certifikat @*fri*CA iz brskalnika (ne osebnega!) in naloži [novega](http://lkrv.fri.uni-lj.si/africa/dokumenti/ca.crt).

**Comment by blazkablatnik** [comment]
Ja, je del kode, ki glede na path ugotovi na kateri strani smo (ugotovi abecedo+mode(=read/write)+diff(easy/medium/hard). Nekako rabm vedt katero funkcijo prozit za tipke iz tipkovnice. Sm predpostavla, da bojo strani vedno mele isti path? Je na devel drugace zrutan? 

Tisto napako, ki vrze za imageRightFlag je pa mislm da samo to da ne more dt listnerja gor (vsa koda je v enm .js fajlu. Bi blo treba jo malo razbit in uredit) ampak ne upliva na drugo delovanje.

**Comment by jaanos** [comment]
Če prav razumem, gre za to stvar:

https://github.com/63150055/crypto-portal/blob/957aca3b333ef61a75f73723790d31067105aa0f/static/js/alphabet/alphabet.js#L2122-L2142

Problem je v tem, da bo pot na strežniku izgledala npr. tako:

https://lkrv.fri.uni-lj.si/crypto-devel/alphabet/greek/read/medium/

(torej vmes je še člen `crypto-portal` oziroma `crypto-devel`). Mislim, da bi moralo bit dovolj, da se preverjajo zadnji trije členi.

**Comment by jaanos** [comment]
Ok, tale zadnji commit popravi zadevo. Bi bilo sicer bolje, če bi bila ta informacija v bazi, kajti trenutna koda se zanaša na to, da imamo samo trenutne tri abecede.
